zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/gateway/StandaloneGatewayAuthenticationNoneIT.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.it.gateway;
+
+import io.camunda.application.Profile;
+import io.camunda.zeebe.gateway.impl.configuration.AuthenticationCfg.AuthMode;
+import io.camunda.zeebe.qa.util.cluster.TestGateway;
+import io.camunda.zeebe.qa.util.cluster.TestStandaloneGateway;
+
+public class StandaloneGatewayAuthenticationNoneIT
+    extends GatewayAuthenticationNoneAbstractIT<TestStandaloneGateway> {
+
+  private final TestStandaloneGateway zeebe =
+      new TestStandaloneGateway()
+          .withAdditionalProfile(Profile.IDENTITY_AUTH)
+          .withGatewayConfig(
+              cfg -> {
+                final var auth = cfg.getSecurity().getAuthentication();
+                auth.setMode(AuthMode.NONE);
+              })
+          .withProperty("camunda.identity.issuerBackendUrl", getKeycloakRealmAddress())
+          .withProperty("camunda.identity.audience", ZEEBE_CLIENT_AUDIENCE);
+
+  @Override
+  protected TestGateway getZeebe() {
```

Comment:
🔧 You can provide the generics here to be more explicit
```suggestion
  protected TestGateway<TestStandaloneGateway> getZeebe() {
```